build_data/web.xml

```diff
@@ -596,36 +596,25 @@
                so the entrypoint script can deactivate Cors settings if DISABLE_CORS is set -->
     <!-- CORS_START -->
     <filter>
-      <filter-name>CorsFilter</filter-name>
-      <filter-class>org.apache.catalina.filters.CorsFilter</filter-class>
-      <init-param>
-        <param-name>cors.allowed.origins</param-name>
-        <param-value>*</param-value>
-      </init-param>
-      <init-param>
-        <param-name>cors.allowed.methods</param-name>
-        <param-value>GET,POST,HEAD,OPTIONS,PUT</param-value>
```

Comment:
Is there a reason why this is rollback from https://github.com/kartoza/docker-geoserver/pull/517?  It brokes some workflow?

Reply:
This breaks the image, tried to build it locally and it was erroring with those changes.

Reply:
and the tests do not pass with those changes

Reply:
Wow I am surprised.  The param `cors.allowed.methods` was enabled for 2.19.0, for us, without any issues.

However we did not have `cors.support.credentials` and `cors.preflight.maxage` so they looked more suspect.

The rest of the changes, you simply changed the indentation so should not matter.

Reply:
I will try another build without cors.preflight.maxage and see if it works. By the way web.xml is completely customizable as a run time config so this doesn't break the image